pkg/constant/const.go

```diff
@@ -30,5 +30,6 @@ const (
 )
 
 const (
-	LastMsgIdUrl = "/admin/v2/persistent/%s/%s/%s/lastMessageId"
+	LastMsgIdUrl   = "/admin/v2/persistent/%s/%s/%s/lastMessageId"
+	ConsumerClosed = "consumer closed"
```

Comment:
go only have package name, may be `ErrorMsgConsumerClosed` is better?
Anyway, we shouldn't put errorMsg and url const in same const group